pkg/internalapis/csinodetopology/csinodetopology.go

```diff
@@ -16,9 +16,240 @@ limitations under the License.
 
 package csinodetopology
 
+import (
+	"context"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/client-go/tools/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"sigs.k8s.io/vsphere-csi-driver/pkg/csi/service/logger"
+	csinodetopologyv1alpha1 "sigs.k8s.io/vsphere-csi-driver/pkg/internalapis/csinodetopology/v1alpha1"
+	k8s "sigs.k8s.io/vsphere-csi-driver/pkg/kubernetes"
+)
+
 const (
 	// CRDSingular represents the singular name of csinodetopology CRD.
 	CRDSingular = "csinodetopology"
 	// CRDPlural represents the plural name of csinodetopology CRD.
 	CRDPlural = "csinodetopologies"
+	// defaultTopologyCRWatcherTimeoutInMin is the default duration for which
+	// the topology service client will watch on the CSINodeTopology instance to check
+	// if the Status has been updated successfully.
+	defaultTopologyCRWatcherTimeoutInMin = 1
```

Comment:
Since this is local to the package, i guess you can just call this variable `defaultTimeoutInMin` for easy readability and leave the comment as is.

Reply:
Done